# Function: <code>pci_cleanup_aer_uncorrect_error_status</code>

## Status
<b>Regular</b>
<ul>
<li>
<details>
<summary>In <code>4.4.0-21-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071583341888,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer/aerdrv_core.c:60",
      "file": "drivers/pci/pcie/aer/aerdrv_core.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/aer/aerdrv_core.c:broadcast_error_message"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071583341888,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 138
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>4.8.0-22-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071583653888,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer/aerdrv_core.c:60",
      "file": "drivers/pci/pcie/aer/aerdrv_core.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/aer/aerdrv_core.c:broadcast_error_message"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071583653888,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 138
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>4.10.0-19-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071583791520,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer/aerdrv_core.c:55",
      "file": "drivers/pci/pcie/aer/aerdrv_core.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/aer/aerdrv_core.c:broadcast_error_message"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071583791520,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 132
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>4.13.0-16-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071583834672,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer/aerdrv_core.c:55",
      "file": "drivers/pci/pcie/aer/aerdrv_core.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/aer/aerdrv_core.c:broadcast_error_message"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071583834672,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 120
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>4.15.0-20-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584097360,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer/aerdrv_core.c:55",
      "file": "drivers/pci/pcie/aer/aerdrv_core.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/aer/aerdrv_core.c:broadcast_error_message"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584097360,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 120
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>4.18.0-10-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584297104,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:366",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_fatal_recovery",
        "drivers/pci/pcie/err.c:pcie_do_fatal_recovery"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584297104,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 122
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.0.0-13-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584394864,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584394864,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 168
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.3.0-18-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584591168,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584591168,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 168
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.4.0-26-generic-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584728960,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584728960,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 168
    }
  ]
}
```
</details>
</li>
<li>
In <code>5.8.0-25-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>5.11.0-16-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>5.13.0-19-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>5.15.0-25-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>5.19.0-21-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>6.2.0-20-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>6.5.0-9-generic-amd64</code>: Absent ❓
</li>
<li>
In <code>6.8.0-31-generic-amd64</code>: Absent ❓
</li>
</ul>
<b>Arch</b>
<ul>
<li>
<details>
<summary>In <code>5.4.0-26-generic-arm64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446603336496989264,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446603336496989264,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 204
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.4.0-26-generic-armhf</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 3230249908,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 3230249908,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 208
    }
  ]
}
```
</details>
</li>
<li>
In <code>5.4.0-26-generic-ppc64el</code>: Absent ❓
</li>
<li>
<details>
<summary>In <code>5.4.0-24-generic-riscv64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446743936275653622,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446743936275653622,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 164
    }
  ]
}
```
</details>
</li>
</ul>
<b>Flavor</b>
<ul>
<li>
<details>
<summary>In <code>5.4.0-1009-aws-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584677360,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584677360,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 171
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.4.0-1010-azure-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584608592,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584608592,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 168
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.4.0-1009-gcp-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584679120,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584679120,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 168
    }
  ]
}
```
</details>
</li>
<li>
<details>
<summary>In <code>5.4.0-26-lowlatency-amd64</code>: ✅</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```

```json
{
  "name": "pci_cleanup_aer_uncorrect_error_status",
  "collision_type": "Unique Global",
  "inline_type": "No",
  "funcs": [
    {
      "addr": 18446744071584786816,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "external": true,
      "loc": "drivers/pci/pcie/aer.c:377",
      "file": "drivers/pci/pcie/aer.c",
      "inline": "seen, unknown",
      "caller_inline": [],
      "caller_func": [
        "drivers/pci/pcie/err.c:pcie_do_recovery",
        "drivers/pci/pcie/dpc.c:dpc_handler"
      ]
    }
  ],
  "symbols": [
    {
      "addr": 18446744071584786816,
      "name": "pci_cleanup_aer_uncorrect_error_status",
      "section": ".text",
      "bind": "STB_GLOBAL",
      "size": 168
    }
  ]
}
```
</details>
</li>
</ul>

## Differences
<b>Regular</b>
<ul>
<li>
No changes between <code>4.4.0-21-generic-amd64</code> and <code>4.8.0-22-generic-amd64</code> ✅
</li>
<li>
No changes between <code>4.8.0-22-generic-amd64</code> and <code>4.10.0-19-generic-amd64</code> ✅
</li>
<li>
No changes between <code>4.10.0-19-generic-amd64</code> and <code>4.13.0-16-generic-amd64</code> ✅
</li>
<li>
No changes between <code>4.13.0-16-generic-amd64</code> and <code>4.15.0-20-generic-amd64</code> ✅
</li>
<li>
No changes between <code>4.15.0-20-generic-amd64</code> and <code>4.18.0-10-generic-amd64</code> ✅
</li>
<li>
No changes between <code>4.18.0-10-generic-amd64</code> and <code>5.0.0-13-generic-amd64</code> ✅
</li>
<li>
No changes between <code>5.0.0-13-generic-amd64</code> and <code>5.3.0-18-generic-amd64</code> ✅
</li>
<li>
No changes between <code>5.3.0-18-generic-amd64</code> and <code>5.4.0-26-generic-amd64</code> ✅
</li>
<li>
<details>
<summary>Removed between <code>5.4.0-26-generic-amd64</code> and <code>5.8.0-25-generic-amd64</code> ➖</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```
</details>
</li>
</ul>
<b>Arch</b>
<ul>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-26-generic-arm64</code> ✅
</li>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-26-generic-armhf</code> ✅
</li>
<li>
<details>
<summary>Removed between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-26-generic-ppc64el</code> ➖</summary>

```c
int pci_cleanup_aer_uncorrect_error_status(struct pci_dev * dev)
```
</details>
</li>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-24-generic-riscv64</code> ✅
</li>
</ul>
<b>Flavor</b>
<ul>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-1009-aws-amd64</code> ✅
</li>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-1010-azure-amd64</code> ✅
</li>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-1009-gcp-amd64</code> ✅
</li>
<li>
No changes between <code>5.4.0-26-generic-amd64</code> and <code>5.4.0-26-lowlatency-amd64</code> ✅
</li>
</ul>
